spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -471,7 +481,7 @@ def get_data_or_scalar(argument, data_in):
         # build data2 with same shape as data
         data_out = data_in[:, :, :] * 0 + float(argument[0])
     # if conversion fails, it should be a string (i.e. file name)
-    except ValueError:
+    except (ValueError, IndexError):
```

Comment:
It's not clear to me what cases exactly are covered by `IndexError`, and I don't think the existing comment on the line above really explains it.

I would be happier if the `try:` block didn't contain such a complicated statement; really, isn't the intention to only cover the `some_value = float(argument[0])` part? I always get a little paranoid that the other stuff could raise a `ValueError` (since so many things can), and then the `except:` block could be covering up a real error.